contracts/nft-querier/src/msg.rs

```diff
@@ -0,0 +1,34 @@
+use crate::state::Config;
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::Addr;
+use cw_ownable::{cw_ownable_execute, cw_ownable_query};
+
+#[cw_serde]
+pub struct InstantiateMsg {
+    pub factory_contract: Addr,
+}
+
+#[cw_serde]
+pub struct MigrateMsg {}
+
+#[cw_ownable_execute]
+#[cw_serde]
+pub enum ExecuteMsg {
+    UpdateConfig { new_config: Config },
```

Comment:
This also applies here. Since in this context a struct of `Config` is used as an external source, you are at risk of accepting an unvalidated `Addr`.

Normally, in other contracts, we define a separate `UpdateConfig` struct to mitigate this issue.